<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneLayerPropertiesv5</class>
 <widget class="QWidget" name="SceneLayerPropertiesv5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>473</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="scrollSpeed">
     <property name="decimals">
      <number>10</number>
     </property>
     <property name="maximum">
      <double>128.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="parallaxFactor">
     <property name="decimals">
      <number>10</number>
     </property>
     <property name="maximum">
      <double>128.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="height">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="width">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="type">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horizontal Scroll</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertical Scroll</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rotozoom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Basic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Scroll Speed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Parallax Factor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Draw Group</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="drawOrder">
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>width</tabstop>
  <tabstop>height</tabstop>
  <tabstop>type</tabstop>
  <tabstop>drawOrder</tabstop>
  <tabstop>parallaxFactor</tabstop>
  <tabstop>scrollSpeed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
